content/help/faq/traffic-management/ingress-with-no-route-rules.md

```diff
@@ -1,57 +0,0 @@
----
-title: Can I use standard Ingress specification without any route rules?
```

Comment:
Hmmm. The plan has been to keep this kind of content in the FAQ. The ops guide is more about day-to-day operations and troubleshooting.